ui/src/components/user.tsx

```diff
@@ -375,6 +376,7 @@ export class User extends Component<any, UserState> {
                 )}
               </ul>
             </h5>
+            <div>{user.bio}</div>
```

Comment:
Make sure this is markdown rendered, check the example in `comment-node` or `post-listing` for this.